### DISCUSSION TITLE
After upgrade from 5.3 to 5.4, all indexes are faulted

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#15648] [General]
I run Raven in a docker container locally for dev. I had the image set for `:latest` because usually I like to see what is going to happen when an upgrade happens.

This morning, I grabbed latest and every database's indexes are now faulted 
<img width="510" alt="image" src="https://user-images.githubusercontent.com/32247592/212701065-0e342d1d-76ff-482f-ad44-6db3f39fba7b.png">

The error message is that the database schema is expecting 54000 but is actually 50000

For local dev, this isn't the biggest problem. I only have 1 test client so I can just delete every index manually & restart my microservices which will rebuild everything I need. I'm just worried about upgrading the other environments.

Is this something I should worry about for my non-local environments or is this more than likely just something that happened in my local machine?

**Comment by garayx** [comment]
Hi

Can you paste the full error + stack trace?
Usually this error means that you tried to open an existing database (for example 5.4) on older RavenDB build (for example 5.3)

**Reply by ItsRobbAllen** [reply]
I'll have to try to recreate this later as I've already fixed the issues, but I did a `docker-compose pull` which pulled the latest RavenDb (5.4) and my existing 5.3 database did not upgrade properly.

I'm still on 5.3 now as I forced the version in in my `docker-compose.yml` file, but I can try again later & see if I can get you the exact error message.

**Comment by ItsRobbAllen** [comment]
I tried to revert everything & recreate this issue, but I could not. The next upgrade went seamlessly, so this was a one-off issue, it would appear.

Thank you for your help.
